### DISCUSSION TITLE
Omnisearch Indexing Never Stop

### DISCUSSION THREAD

**Discussion by kennygokh** [scambier/obsidian-omnisearch#383] [Q&A]
Hi, may i ask whether anyone encountered the message "Work in progress: indexing files..." that is working so slow. I have tried to clear cache and restart Obsidian but this message still remain even after few hours. Any idea what went wrong?

![image](https://github.com/scambier/obsidian-omnisearch/assets/97268671/d65317be-53a8-4b83-9c6e-93e95ba403c0)

**Comment by scambier** [comment]
There are 2 possibilities:
- Either you have a lot of files (and/or many PDFs and Text Extractor is struggling), so it really takes a long time
- Or there is an unhandled crash somewhere and Omnisearch is stuck in this state without progressing. You can check that in the developer console (ctrl+shift+i)

**Comment by kennygokh** [comment]
Hi @scambier, it looks like it stuck somewhere. Please refer to the screenshot for the error. Apology that I am not able to understand error, hope you can help.

<img width="553" alt="image" src="https://github.com/user-attachments/assets/da5f8190-03a4-487a-955c-812aa7fcda43">

**Comment by kennygokh** [comment]
Hi @scambier do you have any findings or advices?

**Reply by scambier** [reply]
Not sure what can cause this kind of crash here, but I added a quick workaround in the latest release that should let Omnisearch skip over the error and continue its indexing.
